llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -427,6 +428,29 @@ Value *VPInstruction::generate(VPTransformState &State) {
                                    {PredTy, ScalarTC->getType()},
                                    {VIVElem0, ScalarTC}, nullptr, Name);
   }
+  // Count the number of bits set in each lane and reduce the result to a scalar
+  case VPInstruction::PopCount: {
+    Value *Op = State.get(getOperand(0));
+    auto *VT = Op->getType();
```

Comment:
nit: Spell out type if it's not present on the RHS.
```suggestion
    Type *VT = Op->getType();
```

Reply:
Done.